provisioning/transport/http/devdoc/http_requirements.md

```diff
@@ -72,14 +68,14 @@ The `constructor` creates an Http transport object used to communicate with the
 
 **SRS_NODE_PROVISIONING_HTTP_18_008: [** If `forceRegistration` is specified, the registration request shall include this as a query string value named 'forceRegistration' **]**
 
-**SRS_NODE_PROVISIONING_HTTP_18_009: [** `register` shall POST the registration request to 'https://global.azure-devices-provisioning.net/{idScope}/registrations/{registrationId}' **]**
+**SRS_NODE_PROVISIONING_HTTP_18_009: [** `register` shall PUT the registration request to 'https://global.azure-devices-provisioning.net/{idScope}/registrations/{registrationId}/register' **]**
+
+**SRS_NODE_PROVISIONING_HTTP_18_010: [** `register` shall wait for the response to the PUT request. **]**
```

Comment:
>wait [](start = 58, length = 4)

I'm ambivalent on this requirement because 'wait' may have the connotation of actively waiting. I'd rather stick to the simple "calls the callback with __ when ___"